modules/dicom_archive/test/TestPlan

```diff
@@ -20,15 +20,17 @@
 6-  Under 'View Details' Check to make sure all the specific parameters are Populated
     [Manual Testing]
 7-  Under 'View Details' Click on 'Acquisition ID' to view the violated scans (if there are any) if permission 
-'Violated Scans: View all-sites Violated Scans' is granted
+    'Violated Scans: View all-sites Violated Scans' is granted
     [Manual Testing]
-8-  Under 'View Details' Click on 'Show/Hide Files' to view a list of all Dicom Files for the given patient
+8-  Under 'View Details' Click on 'Show/Hide Series' and 'Show/Hide Files' to view a list of all Dicom Series/Files,
+    respectively, for the given patient
     [Manual Testing]
-9- Under 'View Details' Click on 'Show/Hide metadata' to view the details of the metadata file
```

Comment:
as far as I know, the show/hide metadata is not there, can someone @cmadjar @driusan confirm?

Reply:
@MounaSafiHarab I confirmed with Dave. It was removed but the test plan was never updated. Thanks for doing it!!